static/js/place_overview.js

```diff
@@ -22,13 +22,27 @@ import { MainPane, Ranking, Menu, ParentPlace } from "./place_overview.jsx";
 
 let ac;
 
+const Y_SCROLL_LIMIT = 150;
+
 window.onload = () => {
   const urlParams = new URLSearchParams(window.location.search);
   let dcid = urlParams.get("dcid");
   renderPage(dcid);
   initAutocomplete();
+  document.addEventListener("scroll", adjustMenuPosition);
 };
 
+function adjustMenuPosition() {
+  let topicsEl = document.getElementById('topics');
+  if (window.scrollY > Y_SCROLL_LIMIT) {
+    topicsEl.classList.add("fixed");
+    topicsEl.style.top = "20px";
```

Comment:
Could you move this to the css so we don't forget to update this?

It should work to use:
#sidebar-outer.fixed { position: fixed; top: 20px }

Reply:
done